sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -59,7 +59,8 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
   test("show functions") {
     def getFunctions(pattern: String): Seq[Row] = {
       StringUtils.filterPattern(
-        spark.sessionState.catalog.listFunctions("default").map(_._1.funcName), pattern)
+        spark.sessionState.catalog.listFunctions("default").map(_._1.funcName)
+        ++ Seq("!=", "<>", "between", "case"), pattern)
```

Comment:
`Seq("!=", "<>", "between", "case")` appears many times, shall we put it in
```
object ShowFunctionsCommand {
  // operators that do not have corresponding functions.
  // They should be handled in `DropFunctionCommand` as well. 
  val virtualOperators = Seq("!=", "<>", "between", "case")
}
```

Reply:
> `Seq("!=", "<>", "between", "case")` appears many times, shall we put it in
> 
> ```
> object ShowFunctionsCommand {
>   // operators that do not have corresponding functions.
>   // They should be handled in `DropFunctionCommand` as well. 
>   val virtualOperators = Seq("!=", "<>", "between", "case")
> }
> ```

Change many times I change name to FunctionsCommand and replace  `Seq("!=", "<>", "between", "case")`


```
object FunctionsCommand {
  // operators that do not have corresponding functions.
  // They should be handled in `CreateFunctionCommand`, `DescribeFunctionCommand`,
  // `DropFunctionCommand` and `ShowFunctionsCommand`
  val virtualOperators = Seq("!=", "<>", "between", "case")
}

```